R/cluster.R

```diff
@@ -84,7 +84,8 @@ generateClusterConfig <- function(fileName, ...){
       ),
       rPackages = list(
         cran = vector(),
-        github = vector()
+        github = vector(),
+        authToken = NULL
```

Comment:
Yes, the token is only for github. Also, I changed authToken to githubAuthToken and pushed it out.